packages/shell-api/src/helpers.ts

```diff
@@ -713,6 +721,10 @@ export type ShardingStatusResult = {
   autosplit: {
     'Currently enabled': 'yes' | 'no';
   };
+  /** Available from 7.0.0 */
+  automerge: {
+    'Currently enabled': 'yes' | 'no';
```

Comment:
Related: is there a documented way of disabling this? Closest is https://www.mongodb.com/docs/manual/reference/method/sh.disableAutoMerger/ but this is collection-specific so not relevant for this.

Reply:
Does https://www.mongodb.com/docs/manual/reference/method/sh.stopAutoMerger/#mongodb-method-sh.stopAutoMerger do the right thing?

Reply:
oh yes! reasonable naming was just a bit of confusing naming when I was looking at it with the context of `disableAutoSplit` which does change things globally